spec/latest/index.bs

```diff
@@ -238,6 +238,30 @@ after reading the values array into memory, it should be interpreted as boolean
 For a given [[#key_format]], all named binary arrays for that format shall have a
 corresponding name in `data_types`.
 
+The following strings shall be used to describe data types:
+: "uint8"
+:: unsigned 8-bit integer
+: "uint16"
+:: unsigned 16-bit integer
+: "uint32"
+:: unsigned 32-bit integer
+: "uint64"
+:: unsigned 64-bit integer
+: "int8"
+:: signed 8-bit integer
+: "int16"
+:: signed 16-bit integer
+: "int32"
+:: signed 32-bit integer
+: "int64"
+:: signed 64-bit integer
+: "float32"
+:: IEEE binary32 floating point number
+: "float64"
+:: IEEE binary64 floating point number
+: "bool"
+:: Boolean number (however that is represented in the host language)
```

Comment:
`complex[float32]`
Array of `n` `complex[float32]` values is stored as an array of `n*2` `float32` values.  The real part of the `i`'th value is stored at index `i/2` and the imaginary part of the `i`'th value is stored at index `(i/2) + 1`.

Reply:
I included the language (although instead of division I say to look at positions 2i and 2i+1)

Reply:
Yes indeed---thanks for correcting my bad math!